packages/utils/src/period/getDefaultPeriod.js

```diff
@@ -16,7 +16,7 @@ export const getDefaultPeriod = () => {
   const startDateToUse =
     EARLIEST_DATA_DATE > minimumStartDate ? EARLIEST_DATA_DATE : minimumStartDate;
   const startDate = startDateToUse.date(1);
-  const endDate = utcMoment().endOf('month');
+  const endDate = utcMoment();
```

Comment:
Nice! One small request, given that we've deviated from the `getDefaultPeriod()` behaviour in web-config-server, can we add a comment either here or in that function explaining the difference. I think it's pretty easy for people to get confused otherwise 👍 

Even better could add a `@deprecated` [jsdoc](https://jsdoc.app/tags-deprecated.html) to the one in web-config-server!